Composer/packages/tools/language-servers/language-understanding/src/matchingPattern.ts

```diff
@@ -177,7 +177,9 @@ export function getSuggestionRoles(luisJson: any, suggestionEntityTypes: string[
         luisJson[entityType].forEach(entity => {
           if (entity.roles !== undefined && entity.roles.length > 0) {
             entity.roles.forEach(role => {
-              suggestionRolesList.push(role);
+              if (role.length > 0) {
```

Comment:
If `role` is a string, we could just:

```suggestion
              if (role) {
```

Or even,

```js
role && suggestionRolesList.push(role);
```